doc/tutorials/walkthrough_inputs.rst

```diff
@@ -353,6 +353,13 @@ Now we need to specify some **settings** that define fundamental reactor
 parameters, as well as modeling approximation options. For this, we make a
 **settings file**, called ``anl-afci-177.yaml``.
 
+First we need to supply some metadata.
+
+.. literalinclude:: ../../armi/tests/tutorials/anl-afci-177.yaml
+    :language: yaml
+    :start-after: begin-metadata
+    :end-before: begin-settings
+
```

Comment:
If you try to delete this metadata section will the settings load or is this actually required as a part of the settings file?

Reply:
For me, ARMI will not run without the metadata entry. 

![image](https://user-images.githubusercontent.com/21959600/127205533-6a070e4e-097b-472e-8812-c2ed928c282c.png)

Once I put the metadata entry in there, it runs with no problems. Do you think it should not be required? If so, perhaps we should check on another machine as well.

Reply:
Okay, thanks! I forgot about this section in the settings being required so this helps.